app/models/container_image.rb

```diff
@@ -36,6 +36,7 @@ class ContainerImage < ApplicationRecord
   after_create :raise_creation_event
 
   def full_name
+    return docker_id if image_ref && image_ref.start_with?(DOCKER_PULLABLE_PREFIX)
```

Comment:
@enoodle how come the tests were not failing before? We're not covering this with a test?